packages/web3-bzz/package.json

```diff
@@ -12,7 +12,7 @@
         "build": "rollup -c",
         "dev": "rollup -c -w",
         "test": "jest",
-        "dtslint": "dtslint types"
+        "dtslint": "dtslint types --onlyTestTsNext"
```

Comment:
Could you add the dtslint command to the package.json in the root folder with lerna? :-)

Reply:
as discuss on telegram i have a PR currently pending which will hopefully be merged by microsoft soon - https://github.com/Microsoft/dtslint/pull/170 will do this after